server/sonar-web/src/main/js/apps/organizations/components/forms/RemoveMemberForm.js

```diff
@@ -50,7 +54,7 @@ export default class RemoveMemberForm extends React.PureComponent {
     this.setState({ open: false });
   };
 
-  handleSubmit = (e: Object) => {
+  handleSubmit = (e /*: Object */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)